src/styles/block/body.scss

```diff
@@ -0,0 +1,7 @@
+body {
+  box-sizing: border-box;
```

Comment:
you already use it in `*`. Remove it from body

Reply:
done